manifests/storage.yaml

```
@@ -13,36 +13,25 @@ spec:
   selector:
     matchLabels:
       k8s-app: hostpath-provisioner
-
   template:
     metadata:
       labels:
         k8s-app: hostpath-provisioner
-
     spec:
       containers:
         - name: hostpath-provisioner
-          image: kjackal/hostpath-provisioner:latest
+          image: balchu/hostpath-provisioner:1.1.1
           env:
             - name: NODE_NAME
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-
-            - name: PV_DIR
+            - name: NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: DEFAULT_PV_DIR
```

i'd suggest reverting this; see above

```suggestion
            - name: DEFAULT_PV_DIR
```

---

Ok 👍